### PR TITLE
Try to fix CI failure since #1050

### PR DESCRIPTION
**Pull request by gjsjohnmurray**
beta workflow failed when updating my fork:
"Unable to resolve action `softprops/action-gh-release@v0`, unable to find version `v0`"

### CODE REVIEW

**Comment by daimor** [comment]
did not notice version in previous PR
checked now, they have v1

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@ jobs:
           name: ${{ steps.set-version.outputs.name }}.vsix
       - name: Create Release
         id: create-release
-        uses: softprops/action-gh-release@v0
+        uses: softprops/action-gh-release@v0.1.15
```
Comment (daimor) [inline .github/workflows/main.yml:116]:
```suggestion
        uses: softprops/action-gh-release@v1
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -158,7 +158,7 @@ jobs:
           npx vsce package -o ${{ steps.set-version.outputs.name }}.vsix
       - name: Upload Release Asset
         id: upload-release-asset
-        uses: softprops/action-gh-release@v0
+        uses: softprops/action-gh-release@v0.1.15
```
Comment (daimor) [inline .github/workflows/main.yml:161]:
```suggestion
        uses: softprops/action-gh-release@v1
```

**Review by daimor — CHANGES_REQUESTED** [review]

**Comment by isc-bsaviano** [comment]
Sorry about that. Can you make the changes in the prerelease yml file as well?

**Comment by gjsjohnmurray** [comment]
@daimor where are you seeing v1? At https://github.com/marketplace/actions/gh-release the latest is v0.1.15

**Comment by daimor** [comment]
image
And readme mentions v1

**Review by isc-bsaviano — APPROVED** [review]

**Comment by gjsjohnmurray** [comment]
@daimor even though @isc-bsaviano has approved it I'm not being offered the merge option. Please approve it too. Maybe that will help.

**Review by daimor — APPROVED** [review]

**Pull request merged**